sdk/tables/azure-data-tables/samples/async_samples/sample_update_upsert_merge_entities_async.py

```diff
@@ -0,0 +1,219 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: table_samples_client.py
+
+DESCRIPTION:
+    These samples demonstrate the following: creating and setting an access policy to generate a
```

Comment:
idt think this description is correct. Also the file name is not table_samples_client.py

Reply:
I think there are other file names that are incorrecti npother samples too